sormas-api/src/main/java/de/symeda/sormas/api/epidata/EpiDataDto.java

```diff
@@ -147,12 +170,32 @@
 	private YesNoUnknown tickBite;
 	@Diseases({Disease.PLAGUE,Disease.UNDEFINED,Disease.OTHER})
 	private YesNoUnknown fleaBite;
-	@Diseases({Disease.MONKEYPOX,Disease.UNDEFINED,Disease.OTHER})
+	@Diseases({Disease.RABIES, Disease.UNDEFINED, Disease.OTHER})
+	private YesNoUnknown kindOfExposure;
```

Comment:
I'm removing the field from both EpiData and EpiDataDto. 
For EpiDataDto, I'll use a getter to return a calculated value.

I've hit a few hurdles:
1. The field is needed in the dto for annotating Diseases. So, I'm keeping the private field
2. The input is disabled when there's no setter for the field. I've therefore added a setter with empty code body

![image](https://user-images.githubusercontent.com/1176692/67201073-f9cc5b00-f3f4-11e9-8815-ef48cb0041a8.png)

Any suggestions?

Reply:
You could also completely remove it from the DTO and have the logic only in the UI. This means using a field in the form that is not bound to the dto. This field would be visible if any of the "kindOfExposure..." fields is visible (which is defined by their disease annotations).

Reply:
OK. I was going for a data-driven approach for the form. Your idea also sounds good.

Reply:
@MartinWahnschaffeSymeda Kindly point to me an example of a field that is not bound to the dto. I'm currently using an example from Vaadin's official docs. Maybe sormas has an already established way

Reply:
To get a basic idea you can have a look at StatisticsVisualizationComponent. 
Take the example of visualizationSelect:

1. The OptionGroup is created (in your case you could also use a Checkbox or add the ValoTheme.OPTIONGROUP_HORIZONTAL style). The constructor receives a list of the possible values.
2. The field is added to the layout. You can do that with a placeholder for the location similar to EpiDataForm.ENVIRONMENTAL_LOC-
3. A value change listener is added. In your case that listener would show or hide the kindOfExposure fields. In theory you could create a new variant of FieldHelper.setVisibleWhen that takes a vaadin field as source instead of the sourcePropertyId
4. The inital value of the field is set by calculating if any kindOfExposure is not null. The logic would ideally be put into an EpiDataHelper class in the api project - similar to SymptomsHelper.

Hope that helps.